src/test/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/services/GeneratePdfUseOfForceServiceTest.kt

```diff
@@ -9,12 +9,11 @@ import com.microsoft.applicationinsights.TelemetryClient
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.mockito.kotlin.mock
-import org.mockito.kotlin.times
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.verifyNoMoreInteractions
 import org.mockito.kotlin.whenever
 import uk.gov.justice.digital.hmpps.subjectaccessrequestworker.models.DpsService
-import uk.gov.justice.digital.hmpps.subjectaccessrequestworker.models.UserDetail
+import uk.gov.justice.digital.hmpps.subjectaccessrequestworker.models.PrisonDetail
```

Comment:
Don't we still wan the test for the replace username with surname

Reply:
The field where that was done has been removed and there doesn't seem to be any other username replacement calls in the template

Reply:
updated as discussed